modules/custom/cu_js_includes/includes/cu_js_includes.permissions.inc

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * Define site permissions in code.
+ *
+ * Create a secure_permissions_data module directory and place this function
+ * in secure_permissions_data.module.
+ *
+ * @param string $role
+ *   The role for which the permissions are being requested.
+ *
+ * @return array
+ *   An array defining all the permissions for the site.
+ */
+function cu_js_includes_secure_permissions($role) {
```

Comment:
See previous comment about granularity of permissions. Additionally, do we need to rebuild secure permissions during install or is that taken care of for us?

Reply:
Secure permissions is called every time a module is enabled.